src/main/java/com/iexec/blockchain/tool/ChainConfig.java

```diff
@@ -47,6 +47,9 @@
     @Value("${chain.is-sidechain}")
     private boolean isSidechain;
 
+    @Value("${chain.start-block-number}")
```

Comment:
This information in only needed to subscribe to Deal events. It's better to add it later if that part is migrated.

Reply:
I think this question relates to the discussion we have with @jeremyjams [here](https://github.com/iExecBlockchainComputing/iexec-common/pull/277#discussion_r750342596). It depends on what we want to achieve. Either have a full configuration server or put only shared values into it. I'd prefer the first one as I find it more consistent.

Reply:
I fully agree with Zied, we don't need it now

Reply:
Alright, so let's remove it! :)